pkg/analysis/markerscope/analyzer_test.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package markerscope
+
+import (
+	"testing"
+
+	"golang.org/x/tools/go/analysis/analysistest"
+)
+
+func TestAnalyzerWarnOnly(t *testing.T) {
```

Comment:
We normally have a test for the default configuration, what is the default? Is defaulting of the config implemented?

Reply:
The following is the default configuration. I will add a defaultConfig function.
- Policy: Warn
- OverrideMarkers: empty (use built-in defaults)
- CustomMarkers: empty